src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,80 @@
+<!---{
+    "experiment": true,
+    "component": "amp-experiment"
+  }--->
+<!-- 
+  #### Introduction
+
+  [amp-experiment](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md) allows to perform user experience experiments on an AMP document and collect resulting data.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-img/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+   <!-- #### Setup -->
+  <!--
+    Import the library.
+  -->
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js"></script>
+  <!--
+    Style different variants of the experiment by using `body[amp-x-name-of-experiment="nameOfVariant"`].
+  -->
+  <style amp-custom>
+    body[amp-x-button-color-experiment="yellow"] .test-button{
+      background-color: yellow;
+      color: black;
+    }
+    body[amp-x-button-color-experiment="red"] .test-button{
+      background-color: red;
+    }
+    body[amp-x-button-color-experiment="blue"] .test-button{
+      background-color: blue;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Configure the experiment realized with `amp-experiment` with a JSON object. Use `variants` within each experiment to declare how many users will be part of an experiment variant. The total sum of the variants must be a number <= 100, if the number is < 100, then the default behaviour is used. An user belongs to one of the variants based on a random number generated between 0 and 100. For more details about variants read the official [documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md). Multiple experiments can be run on the same AMP document in parallel with their own sets of variants. One AMP document can have at most one `amp-experiment` element.
```

Comment:
suggestion: "Configure experiments inside the `amp-experiment` inside a JSON object.